src/pages/InsideContents/Quote.tsx

```
@@ -0,0 +1,699 @@
+                    </Paper>
+                  )}
+                  
+                  <Paper elevation={1} sx={{ p: 3, my: 2, bgcolor: theme.palette.background.default }}>
+                    <Typography 
+                      variant="h6" 
+                      sx={{ mb: 2, color: theme.palette.text.secondary }}
+                    >
+                      Detailed Quote Information
+                    </Typography>
+                    <Typography 
+                      dangerouslySetInnerHTML={{ __html: aiResponse }} 
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)